src/main/typescript/node_modules/@atomist/rug/operations/Decorators.ts

```diff
@@ -54,14 +54,18 @@ function MappedParameter(foreignKey: string) {
  */
 
 function ruglike(fn: string, kind: string, msg: string) {
-  return (name: string, description: string) => {
+  return (nameOrDescription: string, description?: string) => {
     // tslint:disable-next-line:ban-types
     return (ctr: Function) => {
       if (typeof ctr.prototype[fn] !== "function") {
         throw new Error(`${msg}`);
       }
-      ctr.prototype.__name = name;
-      ctr.prototype.__description = description;
+      if (description === undefined) {
+        ctr.prototype.__description = nameOrDescription;
+      }else {
```

Comment:
Does tslint not complain about the lack of space between the closing curly brace and the `else`?